app/build.gradle

```diff
@@ -92,6 +92,9 @@ dependencies {
     implementation "android.arch.persistence.room:rxjava2:$roomVersion"
     testImplementation "android.arch.persistence.room:testing:$roomVersion"
 
+    //Location Play Service
+    implementation 'com.google.android.gms:play-services-location:16.0.0'
```

Comment:
This dependency should only be in the playstore flavor

Reply:
Done. Used playStoreImplementation.